tests/charts/test_webserver.py

```diff
@@ -721,6 +721,30 @@ def test_should_add_component_specific_labels(self):
         assert "test_label" in jmespath.search("metadata.labels", docs[0])
         assert jmespath.search("metadata.labels", docs[0])["test_label"] == "test_label_value"
 
+    @parameterized.expand(
+        [
+            (
+                [{"name": "webserver-nodeport", "nodePort": "31000", "port": "8080"}],
+                [{"name": "webserver-nodeport", "nodePort": 31000, "port": 8080}],
+            )
+        ]
+    )
+    def test_nodeport_service(self, ports, expected_ports):
```

Comment:
looks like your params aren't consistent with those provided in expand
but let's see what the tests tell us
i'd also add an entry (in  expand) for when no port is provided

Reply:
> looks like your params aren't consistent with those provided in expand 

@dstandish Not sure I follow-- I used the test_ports_overrides param structure as a reference, and [it looks like CI checks are passing](https://github.com/apache/airflow/actions/runs/3222282528/jobs/5271598232)

> i'd also add an entry (in expand) for when no nodePort is provided

Added and tested locally with success. We'll see if the remote CI checks pass as well

Reply:
all good.

one thing though. 

looking [here](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport), it looks like `name` is not a valid attr for nodeport. probably best to remov that.